### PR TITLE
fix: mainnet-test

### PR DESCRIPTION
**Pull request by joshualyguessennd**

### CODE REVIEW

File: tests-mainnet/Strategy/yearn_vault/valide_data.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "test_parameter": [
```
Comment (Genysys) [inline tests-mainnet/Strategy/yearn_vault/valide_data.json:2]: Why do we have 2 parameter files?

File: abi/ExploitCompVAkroSwap.json [hunk 2]
```diff
@@ -0,0 +1,286 @@
+[
```
[the remark this answers is not in the document]
Comment (joshualyguessennd) [reply abi/ExploitCompVAkroSwap.json:1]: hmm it was include in the main repository I'll add it on the the .gitignore

File: tests-mainnet/Strategy/workflow/conftest.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+def regular_user1(accounts):
+    yield accounts[2]
+
+
+@pytest.fixture(scope="module")
+def vault_savings(env_settings, Contract):
+    yield Contract.from_explorer(os.getenv("VAULTSAVINGV2_PROXY"), as_proxy_for=os.getenv("VAULTSAVINGV2"))
+
+
+# @pytest.fixture(scope="module")
```
[the remark this answers is not in the document]
Comment (joshualyguessennd) [reply tests-mainnet/Strategy/workflow/conftest.py:37]: ok

File: tests-mainnet/Strategy/workflow/conftest.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+# def vault_tricrypto(env_settings, Contract):
```
[the remark this answers is not in the document]
Comment (joshualyguessennd) [reply tests-mainnet/Strategy/workflow/conftest.py:38]: they are not required

File: tests-mainnet/Strategy/yearn_vault/valide_data.json (hunk 1 above)
Comment (joshualyguessennd) [reply tests-mainnet/Strategy/yearn_vault/valide_data.json:2]: for each case I've created 2 parameters because the fied number are differents .

**Review by Genysys — APPROVED** [review]
LGTM!

**Pull request merged**